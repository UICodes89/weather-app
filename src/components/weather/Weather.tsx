import React from 'react'
import Search from "../search/Search";



export default function Weather() {
  const getClass = (data: string) => {
    return data;
  }

  return (
    <div className={`search-box ${true ? getClass('rainy') : ''}`}>
      <h1>Search Weather <span className="text-muted">by city or, geolocation</span></h1>
      <Search />
    </div >
  )
}
