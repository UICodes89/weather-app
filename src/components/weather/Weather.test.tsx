import React from 'react';
import { render, screen } from '@testing-library/react';
import Weather from './Weather';
import renderer from 'react-test-renderer';

test('renders learn react link', () => {
  render(<Weather />);
});


//snapshot testings
test('Weather Component snapshot', () => {
  const tree = renderer.create(<Weather></Weather>).toJSON();
  expect(tree).toMatchSnapshot()
})