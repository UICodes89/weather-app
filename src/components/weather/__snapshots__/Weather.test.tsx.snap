// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Weather Component snapshot 1`] = `
<div
  className="search-box rainy"
>
  <h1>
    Search Weather 
    <span
      className="text-muted"
    >
      by city or, geolocation
    </span>
  </h1>
  <div
    className="search_container"
  >
    <form
      onSubmit={[Function]}
    >
      <input
        className="search-field"
        onChange={[Function]}
        placeholder="Please enter a city name"
        type="text"
        value=""
      />
      <button
        className="search-btn"
        type="submit"
      >
        search
      </button>
      <a
        className="location"
        href="#"
      />
    </form>
  </div>
</div>
`;
